apps/perf-test/tasks/perf-test.ts

```diff
@@ -233,19 +235,24 @@ module.exports = async function getPerfRegressions() {
       throw new Error(`Invalid scenario: ${scenarioName}.`);
     }
     const iterations = iterationsArg || scenarioIterations[scenarioName] || iterationsDefault;
-    // These lines can be used to check for consistency.
-    // Array.from({ length: 20 }, (entry, index) => {
-    scenarios[scenarioName] = {
-      // scenarios[scenarioName + index] = {
-      baseline: `${urlForMaster}?scenario=${scenarioName}&iterations=${iterations}`,
-      scenario: `${urlForDeploy}?scenario=${scenarioName}&iterations=${iterations}`,
-    };
-
-    scenarioSettings[scenarioName] = {
-      iterations,
-    };
+    const renderTypes = scenarioRenderTypes[scenarioName] || DefaultRenderTypes;
+
+    renderTypes.forEach(renderType => {
+      const scenarioKey = `${scenarioName}-${renderType}`;
+      const testUrlParams = `?scenario=${scenarioName}&iterations=${iterations}&renderType=${renderType}`;
+
+      scenarios[scenarioKey] = {
+        baseline: `${urlForDeploy}${testUrlParams}`, // TODO: revert before merging
```

Comment:
This is temporary and will not merge. I did this because master tests do not have latest perf-test script and will not work for re-render cases. Once this change is ready to merge, I need to stage this change to 2 steps:
1. revert tests to only test `mount` and merge this change
2. merge another change that re-enable the tests for `re-render`